example/e2e/deploy-federated-deployment.sh

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script has an end to end steps in deploying a federated deployment
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+TEST_NS="test-namespace"
+KUBEFED_NS="kube-federation-system"
+
+# Prerequisite: a cluster scoped federation deployment with `cluster` and
+# `cluster2` joined as member clusters
+#
+# kubectl config use-context cluster1
+echo "Starting creating a ngnix application in federation cluster."
+
+echo "Creating a test namespace ${TEST_NS}."
+kubectl create ns ${TEST_NS}
+if [[ $? != 0 ]]; then
+    echo "Failed creating ${TEST_NS}."
+    exit 1
+fi 
+
+echo "Creating a ngnix deployment in namespace ${TEST_NS}."
```

Comment:
Did you try `kubectl run nginx --image=nginx --expose=true` or some similar `kubectl run` options to create the deployment and service?

Reply:
The goal is to have an nginx deployment that serves a message out of a configmap, so that an override can be demonstrated.  This precludes using something like you are suggesting.

Reply:
Got it, I read your suggested steps after posting this comment.